src/source-map.ts

```diff
@@ -72,6 +72,7 @@ function createJsIdentitySourcemap(
     generator.addMapping(mapping);
   });
 
+  generator.setSourceContent(sourceUrl, sourceContent);
```

Comment:
Not sure if related yet, but I found this interesting: https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/#toc-xssi

```js
if (response.slice(0, 3) === ")]}") {
    response = response.substring(response.indexOf('\n'));
}
```